src/modules/lending/BucketLender.ts

```diff
@@ -496,6 +496,14 @@ export default class BucketLender {
     return bucketLender.owner.call();
   }
 
+  public async getPositionID(
```

Comment:
lowercase "d" in "ID" to make consistent with other functions (check this please, I think that's right)